robotlib/include/robotlib/proximity_handler.hpp

```diff
@@ -2,15 +2,18 @@
 
 #include <io-interface/ultra_sonic_sensor.hpp>
 
+using namespace iointerface::sensor;
```

Comment:
don't put `using namespace` in header file 

http://www.lonecpluspluscoder.com/2012/09/22/i-dont-want-to-see-another-using-namespace-xxx-in-a-header-file-ever-again/